src/pages/create/forms/FundingForm/DistributionSplitsSection/types.ts

```diff
@@ -0,0 +1,11 @@
+import * as Moment from 'moment'
+
+export type SplitType = 'project' | 'address'
+
+export type AddOrEditSplitFormFields = {
+  projectId: string
+  beneficiary: string
+  percent: number
+  amount: string | undefined
+  lockedUntil: Moment.Moment | undefined | null
```

Comment:
**question** Is this really a date? I thought it was a `number`

Reply:
Yeah definitely not my first pick, but it is workjng for now so afraid to change it much